cmd/datamon/main.go

```diff
@@ -2,8 +2,20 @@
 
 package main
 
-import "github.com/oneconcern/datamon/cmd/datamon/cmd"
+import (
+	"log"
+	"os"
+	"runtime/pprof"
+
+	"github.com/oneconcern/datamon/cmd/datamon/cmd"
+)
 
 func main() {
+	f, err := os.Create("cpu.prof")
+	if err != nil {
+		log.Fatal(err)
+	}
+	_ = pprof.StartCPUProfile(f)
 	cmd.Execute()
+	defer pprof.StopCPUProfile()
```

Comment:
unclear what the intent is here:  either removing the `defer` or placing the `defer`red `Stop` immediately after the `Start` would be clearer.